source/manual/alerts/slo-rate-checks-account.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-account-tech"
+title: GOV.UK Account SLO error rate exceeded 1% in 10 minute window
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The GOV.UK Account team has defined Service Level Objectives (SLOs) for
+reliability of the `account-api`.
+
+You can read more about these in [GOV.UK Accounts Service Level
+Objectives](https://docs.google.com/document/d/1HVXlyy2vdy9U0O-_7Nw8_Oso7VXQeKgJ2_-bi4pKlz0/edit)
+
+## Warning - "HTTP Error on GOV.UK Account has exceeded an acceptable threshold across the last 10 minutes"
```

Comment:
I don't think the `Warning - "..."` is adding anything here, just use `...` directly